### PR TITLE
Password check implemented

### PR DESCRIPTION
**Pull request by kondorski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: password-check-bonus
🏅 Granted 4 XP for @kondorski

**Comment by ziobron** [comment]
✅ CI Passed - Task: password-check
🏅 Granted 8 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

File: homework/password-check/validation.cpp [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+{
+    if (password.length() < 9)
+    {
+        return ErrorCode::PasswordNeedsAtLeastNineCharacters;
+    }
+    if (std::none_of(password.begin(), password.end(), isdigit))
+    {
+        return ErrorCode::PasswordNeedsAtLeastOneNumber;
+    }
+        if (std::none_of(password.begin(), password.end(), ispunct))
```
Comment (nauka-programowania-MA) [inline homework/password-check/validation.cpp:29]: Wrong indentation.

**Review by nauka-programowania-MA — APPROVED** [review]
BEAUTIFUL!!!! I really enjoy your code style.

**Pull request closed without merging**